ngx-fudis/projects/ngx-fudis/src/lib/foundations/spacing/tokens.scss

```diff
@@ -31,3 +30,10 @@ $spacing-input-lg: calc(23rem / var(--fudis-rem-multiplier));
 $spacing-input-checkbox: calc(1.25rem / var(--fudis-rem-multiplier));
 $spacing-input-radio: calc(1.25rem / var(--fudis-rem-multiplier));
 $spacing-input-radio-indicator: calc(0.75rem / var(--fudis-rem-multiplier));
+
+// Dialog size widths
+$spacing-dialog-xs: calc(30rem / var(--fudis-rem-multiplier));
+$spacing-dialog-sm: calc(44rem / var(--fudis-rem-multiplier));
+$spacing-dialog-md: calc(55rem / var(--fudis-rem-multiplier));
+$spacing-dialog-lg: calc(70rem / var(--fudis-rem-multiplier));
+$spacing-dialog-xl: calc(90rem / var(--fudis-rem-multiplier));
```

Comment:
Not necessarily something that needs to be fixed, but I think we should discuss with the team if these naming conventions make sense. As usually term "spacing" refers to margins and paddings between elements, but now these are used for sizes as well. I think they can live in this file, but as 3.0 is already a breaking change, I think we should rename these as well to go with the flow with 3.0.

Reply:
Discussed and renamed/refactored these